about.md

```diff
@@ -2,40 +2,74 @@
 layout: default
 ---
 
-## Why are we doing this?
+## Can I attend?
 
-Remote Hack helps developers and technologists get together to create solutions to Remote working, collaboration, and socialising. To go full-meta, we'll be hacking on tools for Remote hacking.
+Yes.
 
-As social distancing sets in and many of us switch to full-time Remote, this event is a chance to hack on things that make Remote a little easier.
+Even if you've never been to a hack day, or aren't tech-focused, you're absolutely welcome.
 
-## What isn't it?
+# How it works
```

Comment:
Could this be `##` or put at the top of the page? Seems to be a bit visual heirarchy clash

Reply:
Great shout. Couldn't quite put my finger on why it was awkward.  I've raised the "Can I Attend" section to a `#` to it's the same level as How it Works.  Added an `hr` too to separate it out a bit.

Feel free to change it!

![image](https://user-images.githubusercontent.com/51385/88218098-72e43180-cc57-11ea-9faf-cbb358d90163.png)